Bottom: 637903e6c13ffaf37c1e9be7ac68b84cb6b6274e
Top:    c46859f1d1c4298bf793c8183e9e3b431da651a7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 02:16:10 +0000

Revert "gui-top-window.c: no need to check arguments if not used."

This reverts commit 92d4bab703cd0894421bc8c8fe1aa7a67dc1c679.

If arguments are passed to an action which doesn't accept them - it MUST fail.


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index e243cd7..f54ed93 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2195,9 +2195,11 @@ Opens the window which allows editing of the route styles.
 static int
 AdjustStyle(int argc, char **argv, Coord x, Coord y)
 {
+  if (argc > 1)
+    AFAIL (adjuststyle);
+
   ghid_route_style_selector_edit_dialog
     (GHID_ROUTE_STYLE_SELECTOR (ghidgui->route_style_selector));
-
   return 0;
 }
